example/src/KlaviyoReactWrapper.ts

```diff
@@ -1,450 +0,0 @@
-import {
```

Comment:
This file was mostly just a wrapper around Klaviyo SDK functions, which pulled them out of context from where they were called, plus by wrapping all our methods in try catches here it gives the impression that is how the integrating developer should call them... which doesn't seem like a big confidence boost.